library/file.lisp

```diff
@@ -0,0 +1,876 @@
+(coalton-library/utils:defstdlib-package #:coalton-library/file
```

Comment:
Add `:documentation` field to this package with information about trailing slashes in lisp. Make sure it shows up on the doc generator.